web/src/engine/websites/InManga.ts

```diff
@@ -0,0 +1,96 @@
+import { Tags } from '../Tags';
+import icon from './InManga.webp';
+import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../providers/MangaPlugin';
+import { FetchCSS, FetchJSON, FetchRequest } from '../FetchProvider';
+import * as Common from './decorators/Common';
+
+type APIChapters = {
+    data: string;
+}
+
+type APIChapter = {
+   result : {
+        Identification: string,
+        FriendlyChapterNumber: string
+    }[]
+}
+
+@Common.ImageDirect()
+
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('inmanga', `InManga`, 'https://inmanga.com', Tags.Language.Spanish, Tags.Media.Manga, Tags.Source.Aggregator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    private _createPostRequest(uri: string, form: string) {

```

Comment:
No need to use `_` convention anymore to mark private methods.
The `private` keyword in TypeScript is just fine, alternatively use the new ECMAScript private modifier which is `#`.

Reply:
Other methods as well...